config/charts/body-based-routing/README.md

```diff
@@ -37,12 +44,9 @@ The following table list the configurable parameters of the chart.
 | `bbr.image.hub`              | Registry URL where the image is hosted.                                                                           | 
 | `bbr.image.tag`              | Image tag.                                                                                                        |
 | `bbr.image.pullPolicy`       | Image pull policy for the container. Possible values: `Always`, `IfNotPresent`, or `Never`. Defaults to `Always`. |
+| `provider.name`              | Name of the Inference Gateway implementation being used. Possible values: `istio`, `gke`. Defaults to `none`.     |
+| `inference-gateway.name`     | The name of the Gateway. Defaults to `my-inference-gateway`.                                                      |                        
```

Comment:
Better to align this with the default used in the guides.

Reply:
Done